core/scripts/vrfv2/testnet/v2scripts/super_scripts.go

```diff
@@ -273,6 +285,25 @@ func VRFV2DeployUniverse(
 	helpers.PanicErr(err)
 	fmt.Printf("Subscription %+v\n", s)
 
+	if deployVRFOwner {
+		// VRF Owner
+		vrfOwner, err := vrf_owner.NewVRFOwner(vrfOwnerAddress, e.Ec)
+		helpers.PanicErr(err)
+		authorizedSendersSlice := helpers.ParseAddressSlice(nodesMap[model.VRFPrimaryNodeName].SendingKeys[0].Address)
```

Comment:
but why just one key, we need to perform `vrfOwner.SetAuthorizedSenders` for all sending keys